src/index-hindi.html

```diff
@@ -0,0 +1,655 @@
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">#SaveTheInternet Blog</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/">Updates from the SaveTheInternet team</a>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <footer>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 text-center">
+          <p>
+            <a href="https://github.com/netneutrality/" role="button" class="btn btn-sm btn-info">Made with ♥ by folks just like you</a>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+    <style>
+      <!-- got bored of scrolling up all the way -->
+
+      #response-copied { display: none; }
+
+    </style>
+
+      <!-- Modal -->
+      <div class="modal fade" id="sendModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title"> प्रतिक्रिया भेजें।</h4>
+            </div>
+            <div class="modal-body">
+              <label for="responseSubject">
+                <h4>विषय</h4>
+                <input type="text" id="responseSubject" readonly class="form-control"></label>
+              <h4>आप का जवाब</h4>
+              <textarea class="form-control" id="responseTextarea" readonly></textarea>
+              <h4 id="copy-instruction">ऊपर लिखा हुआ टेक्स्ट कॉपी करें<span class="copy-short"> by Right-click->Copy (or <span class="control-str">Ctrl</span>+C)</span> <button id="copyBtn" class="btn btn-primary">हो गया</button></h4>
+              <div id="response-copied"><h4>एक ईमेल लिखें, फिर उसमें कॉपी किया हुआ टेक्स्ट<span class="copy-short"> (<span class="control-str">Ctrl</span>+V)</span> पेस्ट करें</h4>
+              <p class="button-bar">
+                <a href="#" target="_blank" class="btn btn-primary btn" id="gmailSend">Gmail</a>
+                <a href="#" target="_blank" class="btn btn-primary btn" id="yahooSend">Yahoo</a>
+                <a href="#" target="_blank" class="btn btn-primary btn" id="otherSend">Other</a>
+              </p>
+                <p>आप ईमेल भेजने से पहले संपादित कर सकेंगे</p>
+                </div>
+                <p>ईमेल <strong><span class="text-info">advisorfea1@trai.gov.in</span></strong><br />को भेजा जाना चाहिए
+                और कृपया हमारे ईमेल
+                 <strong><span class="text-info" id="bccAddressModal">netneutralityindia@gmail.com</span></strong> को BCC में रखें|<br />
+                हम आपकी प्रतिक्रिया ट्राई ने स्वीकार किया है कि यह सुनिश्चित करने की कोशिश करेंगे।</p>
```

Comment:
This seems to be grammatically incorrect. `Ham aapki pratikriya TRAI ne svikar kiya hai ki yah…`